src/Backend/DataSeriesFactory.js

```diff
@@ -0,0 +1,29 @@
+export default class TraceFactory {
+    static getXValuesByPoints(min_energy, max_energy, pointNumber) {
```

Comment:
rename `pointNumber` to `number_of_points`

Reply:
This function is not specific to energy, I suggest to rename arguments to `min_value` and `max_value`